src/gl/special_case.cpp

```diff
@@ -101,11 +112,37 @@ uint32_t gl_special_case(uint32_t primitivetype, uint32_t vertextype, struct nve
 		{
 			VOBJ(tex_header, tex_header, VREF(texture_set, tex_header));
 
-			if((uint32_t)VREF(tex_header, file.pc_name) > 32)
-			{
-				// avoid filtering window borders
-				if(!_strnicmp(VREF(tex_header, file.pc_name), "menu/btl_win_c_", strlen("menu/btl_win_c_") - 1) && VREF(texture_set, palette_index) == 0) current_state.texture_filter = false;
-			}
+      if ((uint32_t)VREF(tex_header, file.pc_name) > 32)
+      {
+        // avoid filtering window borders
+        if (!_strnicmp(VREF(tex_header, file.pc_name), "menu/btl_win_c_", strlen("menu/btl_win_c_") - 1) && VREF(texture_set, palette_index) == 0) current_state.texture_filter = false;
+
+        // don't filter original menu textures in ff7, because it introduces glitches
+        if ((VREF(texture_set, ogl.external)))
```

Comment:
Maybe do the opposite check so you avoid having this huge empty block of nothing?

Reply:
I tried it the other way before and it wasn't cooperating.  It cold have been another issue. i did say it was ugly. will look into this.

Reply:
okay change made.

To fully test this, have anisotropic filtering off, and bilinear filtering ON. This is the setting that revealed the most glitches.   I was unable to detect any more of them in my testing, but that doesn't mean there aren't any.